stable/atlantis/README.md

```diff
@@ -31,13 +31,16 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | `aws.config`                                | Contents of a file to be mounted to `~atlantis/.aws/config`.                                                                                                                                                                                                                                                                                               | n/a                    |
 | `aws.credentials`                           | Contents of a file to be mounted to `~atlantis/.aws/credentials`.                                                                                                                                                                                                                                                                                          | n/a                    |
 | `bitbucket.user`                            | The name of the Atlantis Bitbucket user.,This value should not be defined if Atlantis is not working against Bitbucket repositories.                                                                                                                                                                                                                         | n/a                    |
 | `bitbucket.token`                           | The personal access token for the Atlantis Bitbucket user.,This value should not be defined if Atlantis is not integrated with Bitbucket repositories.                                                                                                                                                                                                     | n/a                    |
 | `bitbucket.secret`                          | Bitbucket Server only: The webhook secret for Bitbucket repositories.                                                                                                                                                                                                                                                                                      | n/a                    |
 | `bitbucket.base_url`                        | Bitbucket Server only: The hostname of your Bitbucket Server installation.                                                                                                                                                                                                                                                                                 | n/a                    |
+| `existingSecret`                            | Secret for TLS authentication expected to contain `tls.crt` and `tls.key`                                                                                                                                                                                                                                                                                | n/a                    |
```

Comment:
Suggested change to description:

The name of a Kubernetes Secret for Atlantis' HTTPS certificate containing the following data items:
- `tls.crt` with the public certificate.
- `tls.key` with the private key.

Just a suggestion.  I don't feel that strongly about it.

Reply:
Maybe change the name to `tlsSecretName`?  If `existingSecret`is a convention or it's too much work to change don't sweat it.